docs/src/main/paradox/release-notes/1.0.md

```diff
@@ -0,0 +1,44 @@
+# Alpakka Kafka 1.0
+
+## Final release of Alpakka Kafka 1.0
+
+Theses release notes contain the need-to-know features and changes for Alpakka Kafka since release 0.22.
+
+### A bit of "history"
+
+Alpakka Kafka has made a long journey. It started off as **reactive kafka** with a first release in January 2015 when Akka Streams were still experimental and created by [Krzysiek Ciesielski, @kciesielski](https://github.com/kciesielski). The team at [SoftwareMill](https://softwaremill.com/) were the first stewards of the project.
+
+In 2016 the Akka team at [Lightbend](https://www.lightbend.com/) (at that time still called *Typesafe*) took over the responsibility to take care of it and started offering commercial support from version 0.16 (released May 2017). The Akka team has participated a lot in the creation of the [Reactive Streams](http://www.reactive-streams.org/) industry standard and [Akka Streams](https://doc.akka.io/docs/akka/current/stream/index.html) has been one of its first implementations. 
+
```

Comment:
Perhaps also mention the start of Alpakka (when you joined) and the rename to Alpakka Kafka, including much improvements and hardening by your team and the community.